ipyleaflet/leaflet.py

```diff
@@ -1395,6 +1395,8 @@ class WidgetControl(Control):
     max_height = Int(default_value=None, allow_none=True).tag(sync=True)
     min_height = Int(default_value=None, allow_none=True).tag(sync=True)
 
+    bg_transparent = Bool(False).tag(sync=True, o=True)
```

Comment:
You can remove `o=True`, this is not an option

Reply:
But will the `bg_transparent` get pulled when`get_options` is called for this 
`this.obj = L.control.widgetcontrol(this.get_options());` ?